jekyll/_cci2/github-trigger-event-options.adoc

```diff
@@ -96,6 +96,13 @@ a| * `opened`
 
 |===
 
+All pipelines triggered by link:https://docs.github.com/en/webhooks/webhook-events-and-payloads#pull_request[pull request events] also have the following xref:variables/#pipeline-values[pipeline values] are also populated:
```

Comment:
All pipelines triggered by link:https://docs.github.com/en/webhooks/webhook-events-and-payloads#pull_request[pull request events] also have the following xref:variables/#pipeline-values[pipeline values]  populated:

Reply:
Alternatively:

For all pipelines triggered by link:[https://docs.github.com/en/webhooks/webhook-events-and-payloads#pull_request[pull](https://docs.github.com/en/webhooks/webhook-events-and-payloads#pull_request%5Bpull) request events] the following xref:variables/#pipeline-values[pipeline values] are also populated:

Reply:
Addressed!